problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,17 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
+function getCountriesSortedByPopulation(arr) {
+    return arr.sort((country1, country2) => {
+        console.log(arr);
+        if (country2.population > country1.population) {
+            console.log(arr)
+        }
+        return Math.floor(country2.population - country1.population)
+    })
```

Comment:
All that's left is to add a chained map call to return an array with only the properties you desire.